documentation/source/QuickstartBasicToolkit.md

```diff
@@ -0,0 +1,245 @@
+# The Basic Skills of SG
+
+Learn the basics of model development with SuperGradients. Researchers and machine learning engineers should start here.
+
+
+<details>
+    <summary>1. Train a Model</summary>
+
+##1. Train a Model
```

Comment:
I dont think we need the sub-title if we have the folding sub-title. 
But even if you wish to keep it, please add a space after the ## , so it will be recognized as a title (in all titles)

Reply:
Fixed https://github.com/Deci-AI/super-gradients/commit/d9199dfa4e657910a027f3acfd768c85b6a8fa28